packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/index.tsx

```diff
@@ -205,6 +270,29 @@ const SliceZone: React.FC<SliceZoneProps> = ({
           setIsDeleteSliceZoneModalOpen(false);
         }}
       />
+      {localLibraries?.length != 0 && (
```

Comment:
A second "=" would not hurt here

Reply:
Done, I copy and paste without checkout the code from the other modal, I fixed it at both places